code/game/objects/items/weapons/misc_items.dm

```diff
@@ -75,10 +75,13 @@
 	if(!is_open)
 		return
 	if(!hidden && I.tool_behaviour != TOOL_SCREWDRIVER && I.w_class == WEIGHT_CLASS_TINY)
+		if(istype(I, /obj/item/disk/nuclear))
+			to_chat(user, "<span class='warning'>You think you're gonna need more than crutches if your employers find out what you just tried to do...</span>")
+			return
 		if(I.flags & ABSTRACT)
 			return
-		if(I.flags & NODROP)
-			to_chat(user, "<span class='notice'>[I] is stuck to your hand!</span>")
+		if(!user.drop_item())
```

Comment:
Personally, I think we should directly unequip it instead of drop_item to avoid weird var/hand stuff
```suggestion
		if(!user.unEquip(I))
```

Reply:
Done!